package.json

```diff
@@ -6,19 +6,24 @@
   "dependencies": {
     "axios": "^0.18.0",
     "dotenv": "^6.1.0",
+    "axios-mock-adapter": "^1.15.0",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.6.0",
     "eslint-config-airbnb": "^17.1.0",
     "history": "^4.7.2",
     "jest": "^23.6.0",
+    "materialize-css": "^1.0.0",
+    "moxios": "^0.4.0",
```

Comment:
Have you used this anywhere?

Reply:
For moxios I had used before the sync on tests but I later opted out. I used materialize css to style the mock article.